### PR TITLE
Rhv ansible

### PR DESCRIPTION
**Pull request by cwilkers**
Base directory and role for RHV environment so far

### CODE REVIEW

**Comment by cooktheryan** [comment]
@cwilkers looks like there is a bit of lint in one of the files https://travis-ci.org/openshift/openshift-ansible-contrib/jobs/235343729

**Comment by cwilkers** [comment]
D'oh! Looks like I forgot to read the style guide...

**Comment by cwilkers** [comment]
Lint trap cleaned.

File: reference-architecture/rhv-ansible/inventory/hypervisors/group_vars/hypervisors [hunk 1]
```diff
@@ -0,0 +1 @@
+ansible_ssh_user: root
```
Comment (cooktheryan) [inline reference-architecture/rhv-ansible/inventory/hypervisors/group_vars/hypervisors]: Maybe place this in ansible.cfg if you can...for example

remote_user = ec2-user

File: reference-architecture/rhv-ansible/playbooks/setup-gluster.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - rhsm
+  vars_files:
+    - vars/vault.yaml
+    - vars/main.yaml
+  roles:
+    - rhsm-subscription
+
+- name: Install and configure Glusterfs prerequisites
+  hosts: hypervisors
+  gather_facts: yes
```
Comment (cooktheryan) [inline reference-architecture/rhv-ansible/playbooks/setup-gluster.yaml]: unless required I would only gather facts once

**Comment by cooktheryan** [comment]
@cwilkers if you have any other commits you want to add and lint this I can review post morning meetings

**Comment by cwilkers** [comment]
@cooktheryan I'll be quiet now, I just mile-stoned getting gdeploy to actually work.

**Comment by cooktheryan** [comment]
Woot will check pr soon.

**Pull request merged**